liquidhaskell.cabal

```diff
@@ -65,6 +65,11 @@ source-repository head
   type:     git
   location: https://github.com/ucsd-progsys/liquidhaskell/
 
+flag strict
```

Comment:
`liquid-platform` and `liquid-fixpoint` use a `devel` flag for the same purpose. Probably it would be better to chose one name to use everywhere, and probably using `devel` will require changing the less scripts.

Reply:
Thanks for the pointer! Renamed `strict` to `devel`.